extensions/machine-learning-services/src/serverConfig/serverConfigManager.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+import { QueryRunner } from '../common/queryRunner';
+import * as constants from '../common/constants';
+import { ApiWrapper } from '../common/apiWrapper';
+import * as utils from '../common/utils';
+
+export class ServerConfigManager {
+
+	/**
+	 * Creates a new instance of ServerConfigManager
+	 */
+	constructor(
+		private _apiWrapper: ApiWrapper,
+		private _queryRunner: QueryRunner,
+	) {
+	}
+
+	/**
+	 * Opens server config documents
+	 */
+	public async openDocuments(): Promise<boolean> {
+		return await this._apiWrapper.openExternal(vscode.Uri.parse(constants.mlsDocuments));
+	}
+
+	/**
+	 * Opens ODBC driver documents
+	 */
+	public async openOdbcDriverDocuments(): Promise<boolean> {
+		if (utils.isWindows()) {
+			return await this._apiWrapper.openExternal(vscode.Uri.parse(constants.odbcDriverWindowsDocuments));
+		} else {
+			return await this._apiWrapper.openExternal(vscode.Uri.parse(constants.odbcDriverLinuxDocuments));
```

Comment:
What about mac?

Reply:
the link is the same for linux and mac

---
In reply to: [362902031](https://github.com/microsoft/azuredatastudio/pull/8796#discussion_r362902031) [](ancestors = 362902031)